BaragonAgentService/src/main/java/com/hubspot/baragon/agent/handlebars/PreferSameRackWeightingHelper.java

```diff
@@ -51,4 +54,89 @@ public CharSequence preferSameRackWeighting(Collection<UpstreamInfo> upstreams,
 
     return "";
   }
+
+  /**
+   *
+   * @param weight
+   * @return a string representing the weight, based on the big decimal weight input
+   */
+  public String getWeight(BigDecimal weight) {
+    if (weight.intValue() == 1) {
+      return "";
+    }
+    if (weight.intValue() == 0) {
+      return configuration.getZeroWeightString();
+    }
+    return String.format(configuration.getWeightingFormat(), weight.intValue());
+  }
+
+
+  public CharSequence preferSameRackWeighting(Collection<UpstreamInfo> upstreams, UpstreamInfo currentUpstream, Options options) {
+    final RackMethods rackHelper = new RackMethodsHelper.RackMethods(upstreams);
+    final List<String> allRacks = rackHelper.generateAllRacks();
+    final BigDecimal totalPendingLoad = rackHelper.getTotalPendingLoad(allRacks);
+    final BigDecimal capacity = rackHelper.calculateCapacity(allRacks);
+    return preferSameRackWeightingOperation(upstreams, currentUpstream, allRacks, capacity, totalPendingLoad, null);
+  }
+
+  /**
+   *
+   * @param upstreams
+   * @param currentUpstream
+   * @param options
+   * @return the weight of the current upstream relative to the current rack such that each upstream carries an equal load.
+   *
+   * Addressing github issue: https://github.com/HubSpot/Baragon/pull/270
```

Comment:
We can remove this, since the commit history points to the PR if someone wants to look it up